region/tourian/main/Metroid Room 4.json

```diff
@@ -357,9 +357,18 @@
       "link": [1, 2],
       "name": "Tank the Damage",
       "requires": [
-        {"metroidFrames": 240}
+        {"metroidFrames": 240},
+        {"or": [
+          "Plasma",
+          "Wave",
```

Comment:
I dont see any reason to include wave. It's trivially easy to open the door if you know how (jump shoot), but wave behaves the same as all other beams besides Plasma. Using Wave to open the door while above seems like a higher skill as well though, especially because it's possible to hit the Metroid, or if you dont prepare and shoot it, youre in the same scenario of needing to jump shoot.